docs/guides/control_plane_protection/md5-session-password.md

```diff
@@ -46,7 +46,7 @@ Example for setting an MD5 password:
     set instance <INSTANCE> protocol bgp peer ipv4 <PEER> <SOURCE> authentication-id <AUTHENTICATION-ID>
     ```
 
-=== "Bird"
+=== "BIRD2/3"
```

Comment:
Nit: For readability, I'd suggest to stick to `Bird 2/3`, if you have strong feelings about `BIRD`, I'd go for `BIRD 2/3`

Reply:
In the documentation, on the website, in the repository, and even in the acronym explanation, it’s always written as BIRD, so I think the capitalized version is better.
I'll leave this open for a few days, so others can share their opinion.